arelle/plugin/validate/ESEF/resources/config.xml

```diff
@@ -5,7 +5,27 @@
     <!-- see ../../../../config/disclosuresystem.xml for full comments -->
 
     <DisclosureSystem
-            names="ESMA RTS on ESEF-2022|ESEF-2022|esef-2022|ESEF|esef"
+         names="ESMA RTS on ESEF-2023|ESEF-2023|esef-2023|ESEF|esef"
+         description="ESMA RTS on ESEF-2023 Validation Checks for Consolidated Financial Statements"
+         validationType="ESEF"
+         exclusiveTypesPattern="EFM|GFM|FERC|HMRC|SBR.NL|EBA|EIOPA|ESEF"
+         blockDisallowedReferences="false"
+         validateFileText="false"
+         contextElement="scenario"
+         />
+
+    <DisclosureSystem
+         names="ESMA RTS on ESEF-2023 Unconsolidated|ESEF-Unconsolidated-2023|esef-unconsolidated-2023|ESEF-Unconsolidated|esef-unconsolidated"
+         description="ESMA RTS on ESEF-2023 Validation Checks for Unconsolidated Financial Statements"
+         validationType="ESEF"
+         exclusiveTypesPattern="EFM|GFM|FERC|HMRC|SBR.NL|EBA|EIOPA|ESEF"
+         blockDisallowedReferences="false"
+         validateFileText="false"
+         contextElement="scenario"
+         />
+
+    <DisclosureSystem
+            names="ESMA RTS on ESEF-2022|ESEF-2022|esef-2022"
```

Comment:
I think we should keep 2022 as the default (`esef` and `esef-unconsolidated` disclosure systems) until the conformance suite for 2023 is released.